concourse/scripts/spec/fixtures/pipecleaner_shellcheck.yml

```diff
@@ -12,6 +12,7 @@ jobs:
               - -e
               - -c
               - |
+                echo "((test))"
```

Comment:
what is the reason for adding this line?

Reply:
This file is used by the `concourse/scripts/pipecleaner_test.go`, it is meant to use our pipecleaner python script to run through and see if there are no errors.

Providing the line above, will help us establish if the pipecleaner is running correctly replacing `\(\(.+\)\)` with `DUMY`, and preventing all kinds of shellcheck errors in other tests.